### DISCUSSION TITLE
Hub aqara + hub generico zigbee

### DISCUSSION THREAD

**Discussion by Moviola79** [indomus/forum#2669] [HUB: Altro (Homey, iHost, SmartThings, Fibaro ecc.)]
buongiorno sono nuovo del forum e mi sto affacciando da poco al mondo della domotica. Vi espongo il mio quesito. Ho un hub M2 di Aqara per la gestione delle valvole termostatiche (sempre di aqara) ed inoltre ho un hub generico,sempre zigbee,con cui tramite l'app Smartlife gestisco alcune lampadine ed interruttori. Vorrei sapere se è possibile integrare nell'app di Aqara anche l'hub generico (ed eventualmente a cascata i vari dispositivi a lui connessi).
Vi ringrazio anticipatamente

**Comment by KC-inDomus** [comment]
no, non puoi. ecco il motivo per cui solitamente si adotta un HUB personale sotto il quale integrare tutto.

**Reply by Moviola79** [reply]
ok ti ringrazio per la risposta. ma eventualmente posso associare all'hub di aqara dispositvi di altri marchi? Tipo prese o lampadine zogbee?

**Reply by KC-inDomus** [reply]
no, come sopra
